rfcs/text/0030/process.yml

```diff
@@ -127,3 +127,26 @@
         Entry point information for a session.
 
         Remote client information such as ip, port and geo location.
+
+    - name: tty
+      level: extended
+      type: object
+      short: Information about the controlling TTY device.
+      description: >
+        Information about the controlling TTY device. If set, the process belongs to an interactive session.
+
+    - name: tty.char_device.major
+      level: extended
+      type: long
+      short: The TTY character device's major number.
+      description: >
+        The major number identifies the driver associated with the device.
+      example: 1
```

Comment:
it might be worth adding this sentence:  
```the character device's major and minor numbers can be algorithmically combined to produce more the familiar terminal identifiers such as "ttyS0" and "pts/0.  For more details see https://www.kernel.org/doc/html/v4.11/admin-guide/devices.html```